tests/README.md

```diff
@@ -0,0 +1,6 @@
+# Tests
+
+## Tags
+Tags are used to filter tests, it can be added as `@<tag name>` in top of the feature file.
```

Comment:
`s/, it can be added as/ and are indicated by/`
`s/in top/at the top`

Also, can't we tag individual tests, not just an entire feature file?

Reply:
tags are just a fancy implementation of a markers, pytest supports `-k regex` on the test name maybe there are other markers not sure

Reply:
From a quick search apparently you can tag individual scenarios.